apps/ledger-live-mobile/e2e/specs/addAccounts/bitcoin.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, device } from "detox";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { loadBleState, loadConfig } from "../../bridge/server";
+import PortfolioPage from "../../models/wallet/portfolioPage";
+import DeviceAction from "../../models/DeviceAction";
+import AccountsPage from "../../models/accounts/accountsPage";
+import AddAccountDrawer from "../../models/accounts/addAccountDrawer";
+import { getElementByText } from "../../helpers";
+
+let portfolioPage: PortfolioPage;
+let deviceAction: DeviceAction;
+let accountsPage: AccountsPage;
+let addAccountDrawer: AddAccountDrawer;
+
+const knownDevice = {
+  name: "Nano X de test",
+  id: "mock_1",
+  modelId: DeviceModelId.nanoX,
+};
+
+describe("Add Bitcoin Accounts", () => {
+  beforeAll(async () => {
+    loadConfig("onboardingcompleted", true);
+    loadBleState({ knownDevices: [knownDevice] });
+
+    portfolioPage = new PortfolioPage();
+    deviceAction = new DeviceAction(knownDevice);
+    accountsPage = new AccountsPage();
+    addAccountDrawer = new AddAccountDrawer();
+
+    await portfolioPage.waitForPortfolioPageToLoad();
+  });
+
+  it("open add accounts from portfolio", async () => {
+    await portfolioPage.openAddAccount();
+  });
+
+  it("add Bitcoin accounts", async () => {
+    await addAccountDrawer.importWithYourLedger();
+    await addAccountDrawer.selectCurrency("bitcoin");
+    // device actions and add accounts modal have animations that requires to disable synchronization default detox behavior
+    await device.disableSynchronization();
+    await deviceAction.selectMockDevice();
+    await deviceAction.openApp();
+    await addAccountDrawer.startAccountsDiscovery();
+    await expect(getElementByText("Bitcoin 2")).toBeVisible();
```

Comment:
Just for my understanding, where is this account coming from? Are you setting it in the test somehow? Since you are using a blank `onbaordingCompleted` app json I'm confused where it's coming from

Reply:
it's the default account that is generated when scanning accounts with `MOCK=1` (1 is a "seed" that will determinimistically rule the account resulting of the process. `genAccount` and `Prando` lib under the hood)